yoti-sdk-parent/pom.xml

```diff
@@ -79,22 +79,24 @@
     <maven.compiler.target>${supported.java.version}</maven.compiler.target>
     
     <!-- sdk dependency versions -->
-    <slf4j.version>1.7.36</slf4j.version>
+    <slf4j.version>2.0.6</slf4j.version>
     <bouncy.castle.version>1.70</bouncy.castle.version>
-    <jackson.version>2.13.3</jackson.version>
-    <jackson-databind.version>2.13.3</jackson-databind.version>
-    <protobuf-java.version>3.21.2</protobuf-java.version>
-    <httpmime.version>4.5.13</httpmime.version>
+    <jackson.version>2.14.2</jackson.version>
+    <protobuf-java.version>3.21.12</protobuf-java.version>
+    <httpmime.version>4.5.14</httpmime.version>
     
     <!-- spring versions -->
-    <spring-framework.boot.version>2.7.1</spring-framework.boot.version>
-    <spring-framework.version>5.3.21</spring-framework.version>
-    <spring-framework.security.version>5.7.2</spring-framework.security.version>
+    <!-- Spring Framework 6.0.x requires JDK 17+   -->
+    <spring-framework.version>5.3.25</spring-framework.version>
+    <!-- Spring Framework Boot 3.0.x requires JDK 17+   -->
+    <spring-framework.boot.version>2.7.8</spring-framework.boot.version>
+    <!-- Spring Framework Security 6.0.x requires JDK 17+   -->
+    <spring-framework.security.version>5.8.1</spring-framework.security.version>
```

Comment:
Best to use springboot bom to define spring-related versions imo:
```xml
<dependencyManagement>
  <dependencies>
    <dependency>
      <groupId>org.springframework.boot</groupId>
      <artifactId>spring-boot-dependencies</artifactId>
      <version>${springboot.version}</version>
      <type>pom</type>
      <scope>import</scope>
    </dependency>
  </dependencies>
</dependencyManagement>
```
(and then declare all spring direct dependencies without a version under `dependencies`)

Unless ofc there's a particular need to override a dependency (but even then the version should be specified in the `dependencyManagement` section and the dependency declared without a version -- so it inherits the one define in the `dependencyManagement`)